FFXIVClientStructs/FFXIV/Component/GUI/AtkTooltipManager.cs

```diff
@@ -37,6 +37,7 @@ public struct AtkTooltipInfo {
 
     [FieldOffset(0x8)] public StdMap<Pointer<AtkResNode>, Pointer<AtkTooltipInfo>> TooltipMap;
     [FieldOffset(0x18)] public AtkStage* AtkStage;
+    [FieldOffset(0x14C)] public byte Flag14C; // Allows AddonItemDetail to be shown with Flag14C |= 2.
```

Comment:
Preferably don't name flags with their offset as the name and instead do `Flag1`, `Flag2` and so on